karspexet/economy/views.py

```diff
@@ -11,3 +12,16 @@ def overview(request):
         'shows': shows,
         'user': request.user,
     })
+
+@staff_member_required
+def show_detail(request, show_id):
+    show = Show.objects.get(pk=show_id)
+    taken_seats = [ticket.seat_id for ticket in Ticket.objects.filter(show=show).all()]
+    [coverage] = Show.ticket_coverage(show)
```

Comment:
What's this? I don't get it.

Reply:
It's identical to `coverage = Show.ticket_coverage(show)[0]`